components/Home.vue

```diff
@@ -4,7 +4,7 @@
     padding>
     <div class="column items-center">
       <div v-if="loading">
-        Loading your new site...
+        {{showLoading()}}
```

Comment:
This is a little odd ... I think we're conflating two paradigms by calling a function from a template. We should either use the imperative `$q.loading` API instead of setting and clearing the `loading` flag (and just remove that flag) OR we should use quasar's inner-loading component stuff:

https://v1.quasar-framework.org/vue-components/inner-loading

```html
      <q-inner-loading :showing="loading">
        <q-spinner-gears size="50px" color="primary" />
      </q-inner-loading>
```

If we want it to be full screen, then we should go with the first option (call the imperative API where we're setting the loading flag and remove it and remove the HTML stuff because it's no longer necessary).

Reply:
Refactored this to make more sense.